modules/gitops-default-permissions-of-an-argocd-instance.adoc

```diff
@@ -2,13 +2,13 @@
 //
 // * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
 
-:_content-type: PROCEDURE
+:_mod-docs-content-type: PROCEDURE
 [id="default-permissions-of-an-argocd-instance.adoc{context}"]
 = Default permissions of an Argocd instance
```

Comment:
```suggestion
= Default permissions of an Argo CD instance
```

Reply:
@tmalove Thanks for the feedback. Good catch! Addressed it.